vcd/datasource_vcd_solution_landing_zone.go

```diff
@@ -0,0 +1,143 @@
+package vcd
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dsSlzChildComponent(title string) *schema.Schema {
+	return &schema.Schema{
+		Type:        schema.TypeSet,
+		Required:    true,
+		Description: "",
```

Comment:
Description missing. Maybe something like `"Details of " + title` or something like that.

Reply:
Indeed. Fixed.